### ISSUE TITLE
Asyncronous callback after trackPage/trackEvent

### ISSUE THREAD

**Issue by jimilucio** [revolunet/angular-google-analytics#155]
Hi everyone, 
exists a way to set a callback function after the trackPage/trackEvent?

L.

**Comment by Toxantron** [comment]
No. What would be the use case for this? Could you elaborate little on your
requirements?
On Apr 29, 2016 17:09, "Luciano" <notifications@github.com> wrote:

Hi everyone,
exists a way to set a callback function after the trackPage/trackEvent?

L.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/revolunet/angular-google-analytics/issues/155>

**Comment by jimilucio** [comment]
Yes, we have a page with redirect to the other app, and we would monitoring this event.

```
myFunctionRedirect(){
 Analytics.trackEvent('button', 'click', 'downloadATR');
 window.location.href = 'urlOtherApp';
}
```

The redirect after the trackEvent remove the previous call to the analytics service of the current page.
